dfir_rs/src/scheduled/graph.rs

```diff
@@ -339,16 +396,13 @@ impl<'a> Dfir<'a> {
             }
         }
 
-        // The stratum we will stop searching at, i.e. made a full loop around.
-        let mut end_stratum = self.context.current_stratum;
-
         loop {
+            // TODO(mingwei): logged stratum off by 1?
```

Comment:
Will this TODO be a problem in the interim? Should you put in a check here and panic if broken?

Reply:
Oh this comment is wrong actually, will remove